{"name": "test_guest_can_go_to_login_page", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.WebDriverException: Message: 'chromedriver' executable needs to be in PATH. Please see https://sites.google.com/a/chromium.org/chromedriver/home", "trace": "E   FileNotFoundError: [WinError 2] Не удается найти указанный файл\n\nDuring handling of the above exception, another exception occurred:\nE   selenium.common.exceptions.WebDriverException: Message: 'chromedriver' executable needs to be in PATH. Please see https://sites.google.com/a/chromium.org/chromedriver/home"}, "start": 1637689507754, "stop": 1637689507754, "uuid": "201a9ba1-0183-40d2-8ccf-e90fe039de89", "historyId": "ce89454eb32d81da7abe3d78f79293b7", "testCaseId": "afba63ba4ada9a3044b9179f4835c21a", "fullName": "test_login_page#test_guest_can_go_to_login_page", "labels": [{"name": "tag", "value": "runik"}, {"name": "suite", "value": "test_login_page"}, {"name": "host", "value": "LAPTOP-Q8IQHMO1"}, {"name": "thread", "value": "26356-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_login_page"}]}